test/images/directx-device-plugin/LICENSE

```diff
@@ -0,0 +1,202 @@
+                                 Apache License
```

Comment:
why do we need a separate LICENSE file?

Reply:
> why do we need a separate LICENSE file?

@aarnaud Would it be ok for you to put the DirectX device manager also under the Kubernetes license based on the CNCF CLA?

Reply:
> > why do we need a separate LICENSE file?
> 
> @aarnaud Would it be ok for you to put the DirectX device manager also under the Kubernetes license based on the CNCF CLA?

Then there would be no need for a separate LICENSE file.

Reply:
yes not problem, 

@thomacos you can do a PR or I can do this week if I have time.

Reply:
Thanks, that's great!

It seems to me that from the legal perspective it might be best if you create the PR since you wrote the code. Alternatively @Kerstin08 or I can do that.

However, before doing this I suggest the following:
Let's wait for a recommendation from the Kubernetes team, see https://github.com/kubernetes/kubernetes/pull/93948#issuecomment-674574752.

Personally I would love to see the DirectX device plugin included in the Kubernetes project, even if it is not included in Kubernetes itself but only min the e2e test, for two reasons: First because it tests GPU access, not only a fake device plugin. Secondly would also indirectly continue testing and maintaining the DirectX device plugin in the future.